bugbug/models/bug_type_classification.py

```diff
@@ -0,0 +1,191 @@
+            bug_features.patches(),
+            bug_features.landings(),
+            bug_features.blocked_bugs_number(),
+            bug_features.ever_affected(),
+            bug_features.affected_then_unaffected(),
+            bug_features.product(),
+            bug_features.component(),
+        ]
+
+        cleanup_functions = [
+            feature_cleanup.url(),
+            feature_cleanup.fileref(),
+            feature_cleanup.synonyms(),
+        ]
+
+        self.extraction_pipeline = Pipeline(
+            [
+                (
+                    "bug_extractor",
+                    bug_features.BugExtractor(feature_extractors, cleanup_functions),
+                ),
+                (
+                    "union",
+                    ColumnTransformer(
+                        [
+                            ("data", DictVectorizer(), "data"),
+                            ("title", self.text_vectorizer(min_df=0.001), "title"),
+                            (
+                                "first_comment",
+                                self.text_vectorizer(min_df=0.001),
+                                "first_comment",
+                            ),
+                            (
+                                "comments",
+                                self.text_vectorizer(min_df=0.001),
+                                "comments",
+                            ),
+                        ]
+                    ),
+                ),
+            ]
+        )
+
+        if self.single_class is None:
+            if classifier is None:
+                self.clf = OneVsRestClassifier(
+                    xgboost.XGBClassifier(n_jobs=utils.get_physical_cpu_count())
+                )
+        else:
+            self.clf = xgboost.XGBClassifier(n_jobs=utils.get_physical_cpu_count())
+            self.clf.set_params(predictor="cpu_predictor")
+
+
+
+    def get_labels(self):
+        classes = {}
+
+        if self.single_class is None:
+            for bug_id, *bug_labels in labels.get_labels(self.name_file_list_labels):
+                target = np.zeros(self.target_size)
+                for keyword in bug_labels:
+                    if keyword != '':
+                        assert keyword in self.list_labels, f"unexpected category {keyword}"
+                        target[self.list_labels.index(keyword)] = 1
+                classes[int(bug_id)] = target
+            return classes, self.list_labels
+
+        else:
+            for bug_id, category in labels.get_labels(self.name_file_list_labels):
+                assert category in [self.single_class_name, "Other"], f"unexpected category {category}"
+                classes[int(bug_id)] = 1 if category == self.single_class_name else 0
+            return classes, [0, 1]
+
+
+    def get_feature_names(self):
+        return self.extraction_pipeline.named_steps["union"].get_feature_names()
+
+
+    def overwrite_classes(self, bugs, classes, probabilities):
```

Comment:
In this function we'll have to do something similar to https://github.com/mozilla/bugbug/blob/72659c4e38255b2b7bc31cf2525080b4e6cfbb15/bugbug/models/bugtype.py#L115 (again, the strings we use as labels are not available as keywords).
For now, let's just remove this function completely.